k8s/plugin/src/main.rs

```diff
@@ -171,14 +166,16 @@ async fn execute(cli_args: CliArgs) {
                     UpgradeResources::uninstall(&cli_args.namespace).await;
                 }
             },
-            Operations::Upgrade => {
-                UpgradeResources::apply(
-                    cli_args.upgrade_operator_endpoint,
-                    &cli_args.namespace,
-                    cli_args.kube_config_path,
-                    cli_args.timeout,
-                )
-                .await;
+            Operations::Upgrade(resources) => {
+                println!("{}",
+                Cyan.bold().italic().paint("Note : During upgrade if you create single replica volumes then you may loose them"));
```

Comment:
```suggestion
                Cyan.bold().italic().paint("Note: Volumes which make use of a single volume replica instance will be unavailable for some time during upgrade. It is recommended that you do not create new volumes which make use of only one volume replica."));
```

Reply:
Should this be behind an logging interface?
warn("Note: Volumes which make use of a single volume replica instance will be unavailable for some time during upgrade. It is recommended that you do not create new volumes which make use of only one volume replica.");

On another note if these messages are important for the user, maybe we should keep them somewhere common? Would also useful for localization.